### ISSUE TITLE
Inconsistency in creating a new device

### ISSUE THREAD

**Issue by vinith-reddy25** [stanford-oval/genie-sdk#6]
I created a new device, by using following methods:

1) https://wiki.almond.stanford.edu/en/genie-guide/setup

2) https://wiki.almond.stanford.edu/en/genie-guide/skill

3) Synthesize and training by https://github.com/stanford-oval/genie-toolkit/blob/master/doc/tutorial-basic.md

4) https://wiki.almond.stanford.edu/en/genie-guide/deploy

By following these steps, I deployed a skill, it is deployed. But when we tried to deploy a new skill from another laptop using same steps, then it didn't deployed? I didn't know any further steps need to be followed or not? Any developer key access issue or some other issue?

**Comment by sileix** [comment]
no, there are no further steps to follow. can you describe your problem in more detail with log/error messages, and/or a description of how the system behaves?

**Comment by vinith-reddy25** [comment]
There is no error, but the device is not added.
